pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -916,14 +916,14 @@ void GSRendererHW::SwSpriteRender()
 		for (int x = 0; x < w; x += 2)
 		{
 			u32* di = dpa.value(x);
-			ASSERT(*di + 1 == *dpa.value(x + 1)); // Destination pixel pair is adjacent in memory
+			ASSERT(di + 1 == dpa.value(x + 1)); // Destination pixel pair is adjacent in memory
```

Comment:
Unrelated changes?

Reply:
yeah unrelated to the yaml stuff, but the game I test with crashes due to this bug so I had to fix it.